packages/account/src/Sections/Profile/PhoneVerification/phone-verification-page.tsx

```diff
@@ -32,7 +32,11 @@ const PhoneVerificationPage = () => {
                     <Localize i18n_default_text='Phone number verification' />
                 </Text>
             </div>
-            {show_email_verification ? <ConfirmYourEmail /> : <ConfirmPhoneNumber />}
+            {otp_verification.show ? (
```

Comment:
isnt it better we check the is_email_verified here?
so if is_email_verified is true we direct them to` Phone number verificationFlow`
else we show  `email Verification flow`

Reply:
Yes but since the API for email verification is not ready, I'll use this `otp_verification.show` for now just to display the email verification flow. will add the `is_email_verified` here once the API is ready